__docs__/wonder-blocks-tabs/navigation-tabs.stories.tsx

```diff
@@ -49,3 +60,286 @@ export const Default: StoryComponentType = {
+            <ScenariosLayout scenarios={scenarios}>
+                {(props, name) => (
+                    <NavigationTabs {...props} aria-label={name} />
+                )}
+            </ScenariosLayout>
+        );
+    },
+    parameters: {
+        a11y: {
+            config: {
+                rules: [
+                    // Disabling warning: "Element's background color could not
+                    // be determined because it's partially obscured by another
+                    // element" since these examples can cause the horizontal
+                    // scrollbar to show. Color contrast check is enabled for
+                    // other stories (including the AllVariants)
+                    {id: "color-contrast", enabled: false},
+                ],
+            },
+        },
+    },
+};
+
+/**
+ * The following story shows how the component habdles specific scenarios at a
+ * small screen size.
+ */
+export const ScenariosSmallScreen: StoryComponentType = {
+    ...Scenarios,
+    parameters: {
+        ...Scenarios.parameters,
+        viewport: {
+            defaultViewport: "small",
+        },
+    },
+};
+
+const StyledHeader = addStyle("header");
+const StyledDiv = addStyle("div");
+
+/**
+ * Here is an example of the `NavigationTabs` component used within a header.
+ *
+ * Note: To line up the header bottom border with the NavigationTabs underline
+ * styling, a negative vertical margin is set on the `NavigationTabs`.
+ */
+export const HeaderWithNavigationTabsExample: StoryComponentType = {
+    render() {
+        // Putting styles in the component so it shows in the code snippet
+        const headerVerticalSpacing = sizing.size_150;
+        const styles = StyleSheet.create({
+            pageStyle: {
+                backgroundColor: semanticColor.surface.secondary,
+                height: "100vh",
+                width: "100vw",
+            },
+            headerStyle: {
+                backgroundColor: semanticColor.surface.primary,
+                display: "flex",
+                alignItems: "center",
+                flexWrap: "wrap",
+                borderBottom: `1px solid ${semanticColor.border.primary}`,
+                gap: sizing.size_300,
+                padding: `${headerVerticalSpacing} ${sizing.size_300}`,
+            },
+            navigationTabsRoot: {
+                // set margin to negative value of header vertical spacing so
+                // that selected indicator lines up with header border
+                margin: `-${headerVerticalSpacing} 0`,
+            },
+        });
+
+        return (
+            <StyledDiv style={styles.pageStyle}>
+                <StyledHeader style={styles.headerStyle}>
+                    <img
+                        src="/logo.svg"
+                        width="40px"
+                        alt="Wonder Blocks logo"
+                    />
+                    <SingleSelect
+                        placeholder="Placeholder"
```

Comment:
This one needs a label with the new API!

```suggestion
                        aria-label="Dropdown"
                        placeholder="Placeholder"
```